onetimes.yml

```diff
@@ -254,3 +254,22 @@ events:
       Jitsi server on https://conf.scs.koeln:8443/SCS-Tech
       Dial-In: +49-221-292772-611
     location: "https://conf.scs.koeln:8443/SCS-Tech" 
+  - summary: "SIG Standardization/Certification"
+    begin: 2022-11-15 11:05:00
```

Comment:
This should be 2022-11-16 (if the meeting will be on Wednesday as stated in the description)

Reply:
Oops! Thanks for spotting! Will fix!